modules/core/src/test/java/org/apache/ignite/util/GridCommandHandlerClusterByClassTest.java

```diff
@@ -1329,7 +628,7 @@ public void testPrintTimestampAtEndsOfExecution() {
      */
     @Test
     public void testCacheIdleVerify() throws Exception {
-        IgniteEx ignite = startGrids(2);
+        IgniteEx ignite = crd;
 
         ignite.cluster().active(true);
 
```

Comment:
You should reset testOut before second `assertEquals(EXIT_CODE_OK, execute("--cache", "idle_verify"));`

Reply:
Why?